tests/custom_lookups/tests.py

```diff
@@ -12,6 +13,23 @@
 from .models import Author, MySQLUnixTimestamp
 
 
+@contextlib.contextmanager
+def register_lookup(model, lookup):
+    try:
+        if isinstance(lookup, list):
+            for l in lookup:
+                model.register_lookup(l)
+        else:
+            model.register_lookup(lookup)
+        yield model
+    finally:
+        if isinstance(lookup, list):
+            for l in lookup:
+                model._unregister_lookup(l)
+        else:
+            model._unregister_lookup(lookup)
```

Comment:
What about allowing multiple positional `*lookups` instead, it would make the code more readable.

```python
@contextlib.contextmanager
def register_lookup(model, *lookup):
try:
    for lookup in lookups:
        model.register_lookup(lookup)
    yield model
finally:
    for lookup in lookups:
        model._unregister_lookup(lookup)
```